routes/pull_request.js

```diff
@@ -6,6 +6,22 @@ var thunkify = require('thunkify');
 
 module.exports = function(runtime) {
   return function * (detail) {
+    // Only actions that can introduce a bug number need to be handled. This
+    // also helps avoid race conditions in cases where multiple events are
+    // generated at the same time (for example when a PR has an assignee set
+    // at the time of creation, both "opened" and "assigned" events are sent).
+    // Possible actions are listed here:
+    // https://developer.github.com/v3/activity/events/types/#pullrequestevent
+    var relevantActions = {
```

Comment:
Nice, makes sense :) Couldn't remember if this was using babel or anything that would polyfill it.